api/src/test/java/com/basistech/util/LanguageCodeTest.java

```diff
@@ -136,7 +137,7 @@ public void testDataSpotCheck() {
             },
         };
         int[] idResults = {
-            0, 1, DONT_TEST_ID, DONT_TEST_ID, DONT_TEST_ID, LanguageCode.values().length - 1,
+            0, 1, DONT_TEST_ID, DONT_TEST_ID, DONT_TEST_ID, 83,
```

Comment:
Why change?

Reply:
The number of `LanguageCode`s has changed, so the ID for `STANDARD_MALAY` is no longer `LanguageCode.values().length - 1`.